content/hi/docs/reference/glossary/docker.md

```diff
@@ -0,0 +1,18 @@
+---
+title: डोकर (Docker)
+id: docker
+date: 2018-04-12
+full_link: https://docs.docker.com/engine/
+short_description: >
+  डोकर एक सॉफ्टवेयर प्रौद्योगिकी है जो ऑपरेटिंग-सिस्टम-स्तरीय वर्चुअलाइजेशन प्रदान करता है जिसे कंटेनर भी कहा जाता है।
+
+aka:
+tags:
+  - fundamental
+---
+
+डोकर (विशेष रूप से, डोकर इंजन) एक सॉफ्टवेयर प्रौद्योगिकी है जो ऑपरेटिंग-सिस्टम-स्तरीय वर्चुअलाइजेशन प्रदान करता है जिसे {{< glossary_tooltip text="कंटेनर" term_id="container" >}} भी कहा जाता है।
+
+<!--more-->
+
+डोकर लिनक्स कर्नेल के संसाधन अलगाव सुविधाओं का उपयोग करता है जैसे कि cgroups और कर्नेल नेमस्पेस, और एक संघ-सक्षम फ़ाइल सिस्टम जैसे कि OverlayFS और अन्य स्वतंत्र कंटेनरों को एक लिनक्स इंस्टेंस के भीतर चलाने की अनुमति देते हैं, वर्चुअल मशीन (भीएम) को शुरू करने और बनाए रखने के ऊपरी हिस्से से बचते हैं।
```

Comment:
This doesn't read right grammatically.

"इंस्टेंस के भीतर चलाने की अनुमति देते हैं" ->  Verb needs to be singular masculine

"वर्चुअल मशीन (भीएम) को शुरू करने और बनाए रखने के ऊपरी हिस्से से बचते हैं।" -> There needs to be a connector, otherwise it doesn't make sense. 

```suggestion
डोकर लिनक्स कर्नेल के संसाधन अलगाव सुविधाओं का उपयोग करता है जैसे कि cgroups और कर्नेल नेमस्पेस, और एक संघ-सक्षम फ़ाइल सिस्टम जैसे कि OverlayFS और अन्य स्वतंत्र कंटेनरों को एक लिनक्स इंस्टेंस के भीतर चलाने की अनुमति देते हैं, वर्चुअल मशीन (भीएम) को शुरू करने और बनाए रखने के ऊपरी हिस्से से बचते हैं।
```

Reply:
@divya-mohan0209 What connector should I use here please suggest

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix